integration-tests/pkg/collector/collector_docker.go

```diff
@@ -41,6 +41,7 @@ func NewDockerCollectorManager(e executor.Executor, name string) *DockerCollecto
 		"/host/etc:ro":              "/etc",
 		"/host/usr/lib:ro":          "/usr/lib",
 		"/host/sys/kernel/debug:ro": "/sys/kernel/debug",
+		"/etc/stackrox:ro":          "/tmp",
```

Comment:
Consider creating a separate directory for this like `/tmp/collector-test` and use that instead of the full /tmp directory.

Reply:
Done